### ISSUE TITLE
Feature requrest: show histograms

### ISSUE THREAD

**Issue by wl2776** [IENT/YUView#155]
It would be useful to show color histograms of video frames.

**Comment by ChristianFeldmann** [comment]
Yes that would be sweet. However it is a good question where and how to show them. Also: How detailed would they have to be / would they be needed?

**Comment by wl2776** [comment]
My case: show in a separate window, appearing after pressing a hotkey or selecting respective item in a menu. Histogram should show color distribution of current displayed frame and should update on frame changes. If the source video in YUV color space, show Y, U, and V values, if in RGB, then - red, green and blue.
[Vooya](http://www.offminor.de) does this.

**Comment by ChristianFeldmann** [comment]
We are on track with one tiny first step. I want to add bitrate plotting so we will include the QCharts package which would be ablte to also handle this.
